implementations/rust/ockam/ockam_command/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "ockam_command"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+publish = true
```

Comment:
`ockam_command` isn't ready to be published, can you set this to `false` for now please?